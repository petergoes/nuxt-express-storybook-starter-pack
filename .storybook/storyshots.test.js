import Vue from 'vue'
import initStoryshots from '@storybook/addon-storyshots'

initStoryshots()
