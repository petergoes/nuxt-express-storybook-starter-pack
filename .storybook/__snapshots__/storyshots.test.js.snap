// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots AppFooter App footer 1`] = `
<footer>
  
  Visit our website for more documentation : 
  <a
    href="https://nuxtjs.org"
    target="_blank"
  >
    nuxtjs.org
  </a>
</footer>
`;
